uint8 version             
uint8 network_id          
uint8 command             
int8  strength            
uint8 left_proximity      
uint8 right_proximity     
uint8 left_floor          
uint8 right_floor         
int16 acceleration_x      
int16 acceleration_y      
int16 acceleration_z      
    
uint16 light
int8   temperature
uint16 battery

uint8 input_a
uint8 input_b

uint8[6] mac_address